.github/workflows/eco-gotests-integration.yml

```diff
@@ -0,0 +1,43 @@
+name: eco-gotests integration
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
```

Comment:
Can we use ubunutu-latest instead?

Reply:
I like to specify the exact version.  Recently they made `ubuntu-24.04` available and within a couple of days those runners had random timeout failures that didn't affect the 22.04 runners.  At least when you specify you know exactly what environment you're getting.